### PR TITLE
"Road to seniority" abstract

### PR DESCRIPTION
**Pull request by msvrtan**
Please help me abstract this talk: I would love to hear suggestions on the abstract/title but also would appreciate feedback on how interesting you feel the subject is.

I'm preparing a set of new talks for 2019, I'm aware I dont have the time to prepare all of them but as they are 'my babies' it's rather hard to be objective on how interesting a particular topic is.

### CODE REVIEW

File: talks/road-to-seniority.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+### Title:
+Road to seniority
```
Comment (zoka123) [inline talks/road-to-seniority.md:2]: Prepare to have "What defines a senior?" discussion here :)

**Comment by pelshoff** [comment]
Why should attendees come to this talk, and what can they take away from it? Sell it to me baby :D

**Review by HcsOmot — COMMENTED** [review]
This is always a very interesting topic that sparks a lot of (sometimes heated) discussion.

Fact of the matter is that seniors are higher up on the food chain of developers.

I want to know what makes a senior, why that's better than being a mid-level developer, and how do I get there.

Besides the title tag and some prestige, what are, in your opinion, benefits of being a senior developer?
Should developers generally strive to become senior developers, or is it enough to have a 'can do' attitude and a genuine drive for self improvement, which enables them learn new concepts, advance their skills and knowledge both vertically and horizontally?

I sometimes feel we as industry tend to give out 'senior' dev titles too easily, while not clearly communicating what's the next possible step.

Also, people like Uncle Bob, Grady Booch, Michael Hill (GeePawHill on twitter) and numerous others give a whole other meaning to the phrase 'senior developer'.

**Comment by msvrtan** [comment]
@HcsOmot what of this things you think I should put in the abstract? I dont have 6 pages of text to put in (which might than kill the need for the talk as well :) )

File: talks/road-to-seniority.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+### Abstract:
+
+While I will share some examples from my own career, this is not a talk about me. This talk will be heavily based on examples of ideal situation vs reality (theory vs practice), talking about do's and dont's on how to build and improve your career, independently if you already are a senior or not.
```
Comment (DamirSvrtan) [inline talks/road-to-seniority.md]: I wouldn't start with a sentence talking about myself, but rather some motivation.

_The road to seniority is an ill defined path - do you consider yourself a senior, do others consider you, and by what definition? Lets lay out the foundations of what defines a senior and let's walk through examples of ideal situations vs reality (theory vs practice). What are some do's and dont's when improving a software career and how does one really become a senior?_

File: talks/road-to-seniority.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+I would love that you as attendee participate and share your stories with all of us, as I will leave some time at the end of the talk for a short discussion.
```
Comment (DamirSvrtan) [inline talks/road-to-seniority.md]: Let's not make this talk a one way street - I'd love that you as attendee participate and share your stories with all of us, as I will leave some time at the end of the talk for a short discussion.

**Review by DamirSvrtan — APPROVED** [review]
I'd love to hear this talk live!

**Comment by msvrtan** [comment]
Thank you all for reviewing!

**Pull request merged**